### PR TITLE
New policy certificate-otp for R2W in nethgui

### PR DESCRIPTION
**Pull request by stephdl**
Make a new option in Nethgui to display `$openvpn@host-to-net{'AuthMode'}=certificate-otp`

https://github.com/NethServer/dev/issues/6112

### CODE REVIEW

File: root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst [hunk 1]
```diff
@@ -27,6 +27,9 @@ Authentication mode
     * Certificate: choose when creating a net2net configuration
     * User, password and certificate: this is the most secure combination.
       The user must be a system user.
+    * User, one time password and certificate: This policy provides a multi
+      factor of Authentication, the user must install an OTP application on his 
+      mobile phone, the code changes every 30 seconds. The user must be a system-user.
```
Comment (gsanchietti) [inline root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst]:
```suggestion
      factor of Authentication, users must enable 2FA from new Server Manager and  they will be required 
      to provide a temporary verification code (OTP) generated by an application running on their smartphone. 
      The user must be a system-user. 
```

File: root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst [hunk 2]
```diff
@@ -28,8 +28,10 @@ Authentication mode
     * User, password and certificate: this is the most secure combination.
       The user must be a system user.
     * User, one time password and certificate: This policy provides a multi
-      factor of Authentication, the user must install an OTP application on his 
-      mobile phone, the code changes every 30 seconds. The user must be a system-user. 
+      factor of Authentication, users must enable 2FA from new Server Manager
```
Comment (gsanchietti) [inline root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst]:
```suggestion
      factor of authentication, users must enable 2FA from new Server Manager 
```

File: root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst [hunk 3]
```diff
@@ -28,8 +28,10 @@ Authentication mode
     * User, password and certificate: this is the most secure combination.
       The user must be a system user.
     * User, one time password and certificate: This policy provides a multi
-      factor of Authentication, the user must install an OTP application on his 
-      mobile phone, the code changes every 30 seconds. The user must be a system-user. 
+      factor of Authentication, users must enable 2FA from new Server Manager 
+      and  they will be required to provide a temporary verification code (OTP)
```
Comment (gsanchietti) [inline root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst]:
```suggestion
      and they will be required to provide a temporary verification code (OTP) 
```

**Review by gsanchietti — COMMENTED** [review]
A couple of cosmetic fixes more ;)

File: root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst (hunk 1 above)
Comment (stephdl) [reply root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst]: done

File: root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst (hunk 2 above)
Comment (stephdl) [reply root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst]: done

File: root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst (hunk 3 above)
Comment (stephdl) [reply root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst]: done

**Pull request merged**